# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/binutils/sparc64-pagesize.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- binutils-2.41/bfd/elf64-sparc.c.vanilla	2023-12-11 22:41:53.508240050 +0100
+++ binutils-2.41/bfd/elf64-sparc.c	2023-12-11 22:43:30.350231463 +0100
@@ -888,7 +888,7 @@
 #define TARGET_BIG_SYM	sparc_elf64_vec
 #define TARGET_BIG_NAME	"elf64-sparc"
 #define ELF_ARCH	bfd_arch_sparc
-#define ELF_MAXPAGESIZE 0x100000
+#define ELF_MAXPAGESIZE 0x2000
 #define ELF_COMMONPAGESIZE 0x2000
 
 /* This is the official ABI value.  */
